authentik/enterprise/apps.py

```diff
@@ -32,3 +32,9 @@ def check_enabled(self):
         from authentik.enterprise.models import LicenseKey
 
         return LicenseKey.get_total().is_valid()
+
+    def reconcile_install_middleware(self):
```

Comment:
```suggestion
    def reconcile_global_install_middleware(self):
```

Reply:
Also, I don't think this should be here, it only needs to be in the enterprise_audit app